config/environments/production.rb

```diff
@@ -31,7 +31,7 @@
 
   # Disable Rails's static asset server
   # In production, Apache or nginx will already do this
-  config.serve_static_assets = false
+  config.serve_static_assets = true
```

Comment:
On second thought, I'm going to do something like this and update `app.json` (and the documentation).